plugins/base/src/main/kotlin/allModulePage/MultimodulePageCreator.kt

```diff
@@ -48,18 +50,18 @@ class MultimodulePageCreator(
                     val displayedModuleDocumentation = getDisplayedModuleDocumentation(module)
                     val dri = DRI(packageName = MULTIMODULE_PACKAGE_PLACEHOLDER, classNames = module.name)
                     val dci = DCI(setOf(dri), ContentKind.Comment)
-                    val header =
-                        ContentHeader(listOf(linkNode(module.name, dri)), 2, dci, emptySet(), emptySet())
+                    val extraWithAnchor = PropertyContainer.withAll(SymbolAnchorHint(module.name))
+                    val header = linkNode(module.name, dri, DCI(setOf(dri), ContentKind.Main), extra = extraWithAnchor)
                     val content = ContentGroup(
                         children =
                         if (displayedModuleDocumentation != null)
                             DocTagToContentConverter.buildContent(displayedModuleDocumentation, dci, emptySet())
                         else emptyList(),
                         dci = dci,
                         sourceSets = emptySet(),
-                        style = emptySet()
+                        style = emptySet(),
```

Comment:
Am I not seeing here, or is this coma completely unwanted here?

Reply:
rmed